pkg/arvo/app/chat-store.hoon

```diff
@@ -1,6 +1,7 @@
 :: chat-store: data store that holds linear sequences of chat messages
 ::
-/+  *chat-json, *chat-eval, default-agent, verb, dbug
+/-  store=chat-store
```

Comment:
We discussed importing "native" types (and maybe libs) with `*`, and thought we agreed on that? Having `:store` suffixes, inside of the store, feels a little bit silly to me. (Same goes for the other apps.)

Reply:
I've been flipping back and forth on this.  I like having `:store` suffixes because 
1. using the autoname syntax doesn't shadow the type.
2. It preserves a semblance of referential transparency inside the chat context. It's not uncommon to have to make changes to both the -store and -hook and I think preserving the namebindings is useful to prevent unnecessary context switches
3. Types aren't really that hard up for space in most common syntactical situations, so identifier length is not really an issue here.

Reply:
"Each file can get one sur `*` or one lib `*`, after that things get too confusing" seems like a reasonable compromise. Otoh `-lib` suffixes appearing anywhere are a sign that something has gone obviously wrong, and imo a fairly strong argument for a "stacking" model of "a library will frequently subsume the structure by the same name, pull in only one of them" - I don't believe e.g. `inbox-to-configs:store` would be overly unclear compared to `inbox-to-configs:store-lib`.

Reply:
Having libs and matching surs under the same namespace sounds pretty nice, yeah. Lacking support for simultaneous `/-  name=thing` and `/+  name=thing`, doesn't seem too unreasonable to just expose the types with the lib, under the assumption you're gonna need them anyway.